lib/stage-autocompleter.js

```diff
@@ -124,6 +124,53 @@ class StageAutoCompleter {
   getCompletions(editor, session, position, prefix, done) {
     // Empty prefixes do not return results.
     if (prefix === '') return done(null, []);
+
+    // Comments block do not return results.
+    const getMatches = (str, regex) => {
+      regex.lastIndex = 0;
+
+      const matches = [];
+      let match = regex.exec(str);
+
+      while (match) {
+        matches.push(match);
+
+        if (regex.lastIndex === match.index) {
+          regex.lastIndex++;
+        }
+
+        match = regex.exec(str);
+      }
+
+      return matches;
+    };
+    let value = session.getValue();
+    const findStringsWithSlashesRe = /("(?:[^"\\]*|\\[\S\s])*"|'(?:[^'\\]*|\\[\S\s])*')/gm;
+    const stringSlashes = getMatches(value, findStringsWithSlashesRe);
+
+    stringSlashes.forEach((item) => {
+      value = [
+        value.slice(0, item.index),
+        '#'.repeat(item[0].length),
+        value.slice(item.index + item[0].length)
+      ].join('');
+    });
+
```

Comment:
If we have input like this:
```
{x: 'uuu//UU'} // $
```
we need to check if ´//´ is a real start tag of the comment, or it can be a part of a string. I couldn't create a single regex to handle it, therefore i am replace stings like this which contain `//` with `{x: #########} // $` and after it i can find real comments.